docs/reports/cost-summary.md

```diff
@@ -0,0 +1,161 @@
+
+The page uses the standard layout with a breakdown of resource groups in the chart and table. The table has a further breakdown by resource.
+
+<br>
+
+## Resources
+
+The **Resources** page includes a breakdown of cost by resource. This is useful for determining which resources are accruing the most cost.
+
+The page uses the standard layout with a breakdown of resources in the chart and table. Instead of a hierarchy, The table includes columns about the resource location, resource group, subscription, and tags.
+
+<br>
+
+## Commitments
+
+<!-- NOTE: This page is duplicated in the commitment-discounts.md. Please keep both updated at the same time. -->
+
+The **Commitments** page serves 3 primary purposes:
+
+1. Determine if there are any under-utilized commitments.
+2. Facilitate chargeback at a subscription, resource group, or resource level.
+3. Summarize cost savings obtained from commitment-based discounts.
+
+This page uses the standard layout with a breakdown of commitment-based discounts in the chart and table.
+
+In addition to cost and savings KPIs, there is also a utilization KPI for the amount of commitment-based discounts that have been utilized during the period. Low utilization will result in lost savings potential, so this number is one of the most important KPIs on the page.
+
+The chart breaks down the cost of used (utilized) vs. unused charges. Unused charges are split out by commitment type (e.g., reservation, savings plan).
+
+The table shows resource usage against commitment-based discounts with columns for resource name, resource group, subscription, and commitment. Use the table for chargeback and savings calculations.
+
+This page filters usage down to only show charges related to commitment-based discounts, which means the total cost on the Commitments page won't match other pages, which aren't filtered by default.
+
+<br>
+
+## Hybrid Benefit
+
+<!-- NOTE: This page is duplicated in the commitment-discounts.md. Please keep both updated at the same time. -->
+
+The **Hybrid Benefit** page shows Azure Hybrid Benefit (AHB) usage for Windows Server virtual machines (VMs). The page uses the standard filters, but differs with the other sections.
+
+Instead of cost KPIs, the page shows how many VMs are currently enabled and how many vCPUs are used.
+
+There are 3 charts on the page:
+
+1. SKU names and number of VMs currently using less than 8 vCPUs. These are under-utilizing AHB.
+2. SKU names and number of VMs with 8+ vCPUs that are not currently using AHB.
+3. Daily breakdown of AHB and non-AHB usage (excluding those where AHB is not supported).
+
+The table shows a list of VMs that are currently using or could be using AHB with their vCPU count, AHB vCPU count, resource group, subscription, cost and quantity.
+
+<br>
+
+## See also
+
+- [Common terms](./terms.md)
+
+---
+
+## Future considerations
+
+- #11
```

Comment:
Should we add support for MG here?

Reply:
Let's see what customers and the community say. I'd rather avoid spending time there if we can, but if needed, I do think we have some cheap workarounds.